cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -4318,7 +4358,8 @@ search = | tstats `cs_summariesonly_authentication` count from datamodel=Cyences
 | iplocation src \
 | `cs_current_week_login_count(pan:globalprotect)` \
 | `cs_country_login_percentage` \
-| eval cyences_severity = case(percentage_login_from_country==0, "high", true(), "high") \
+| `cs_user_privilege_mapping(user)` \
+| eval cyences_severity = case(percentage_login_from_country==0 or is_privileged_user=="Yes", "high", true(), "high") \
```

Comment:
Critical, maybe, as for all its high.

Reply:
It's reduced from critical to high due to new user login attempt.

Reply:
as discussed in other comment.